google-http-client/src/main/java/com/google/api/client/http/HttpBackOffIOExceptionHandler.java

```diff
@@ -97,6 +97,12 @@ public boolean handleIOException(HttpRequest request, boolean supportsRetry) thr
     try {
       return BackOffUtils.next(sleeper, backOff);
     } catch (InterruptedException exception) {
+<<<<<<< HEAD
```

Comment:
bad merge

Reply:
oops... cant believe this got in here without me noticing.
fixed. ptal.